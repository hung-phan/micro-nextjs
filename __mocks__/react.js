// https://github.com/airbnb/enzyme/issues/1875
// This is to fix the error with React.memo

module.exports = {
  ...require("react"),
  memo: x => x
};
