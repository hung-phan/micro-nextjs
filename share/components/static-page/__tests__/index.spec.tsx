import { shallow } from "enzyme";
import * as React from "react";

import StaticPage from "..";

describe("Component: StaticPage", () => {
  let component;

  beforeEach(() => {
    component = shallow(<StaticPage />);
  });

  it("should render component", () => {
    expect(component).toMatchSnapshot();
  });
});
