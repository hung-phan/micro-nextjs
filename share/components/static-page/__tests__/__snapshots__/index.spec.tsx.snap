// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: StaticPage should render component 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <h1>
        Static Page
      </h1>
      <ForwardRef(LinkComponent)
        href="/"
      >
        <a>
          Back to Home page
        </a>
      </ForwardRef(LinkComponent)>
    </div>
  </div>
</div>
`;
